ui/ducks/domains.js

```diff
@@ -143,17 +142,17 @@ export function initializeDomainSlice() {
   return (dispatch, getState) => {
     const state = getState();
     const chainId = getCurrentChainId(state);
-    const network = CHAIN_ID_TO_NETWORK_ID_MAP[chainId];
-    const networkName = NETWORK_ID_TO_ETHERS_NETWORK_NAME_MAP[network];
-    const ensAddress = networkMap[network];
+    const networkName = CHAIN_ID_TO_ETHERS_NETWORK_NAME_MAP[chainId];
+    const chainIdInt = parseInt(chainId, 16);
```

Comment:
This could return `NaN`, is it worth checking?

Reply:
honestly not sure... I don't check if `parseInt` is returns `NaN` because it's the pattern I've generally seen used elsewhere in extension. For example, see line 151 in this file